docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -218,29 +190,17 @@ config example using the secret you created above.
     < . . . >
 ----
 +
-* If you need to present extra files to any of the services, you can use
-`extraMounts`. For example, when using ceph, you'd need the {rhos_component_storage_file} ceph
-user's keyring file as well as the `ceph.conf` configuration file
-available. These are mounted via `extraMounts` as done with the example
-below.
-* Ensure that the names of the backends (`share_backend_name`) remain as they
-did on {OpenStackShort} {rhos_prev_ver}.
-* It is recommended to set the replica count of the `manilaAPI` service and
-the `manilaScheduler` service to 3. You should ensure to set the replica
-count of the `manilaShares` service/s to 1.
-* Ensure that the appropriate storage management network is specified in the
-`manilaShares` section. The example below connects the `manilaShares`
-instance with the CephFS backend driver to the `storage` network.
-* Prior to adopting the `manilaShares` service for CephFS through NFS, ensure that
-you have a clustered Ceph NFS service created. You will need to provide the
-name of the service as ``cephfs_nfs_cluster_id``.
+* If you need to add extra files to any of the services, you can use `extraMounts`. For example, when using {Ceph}, you can add the {rhos_component_storage_file} Ceph user's keyring file as well as the `ceph.conf` configuration file.
+* Ensure that the names of the back ends (`share_backend_name`) are the same as they were in {OpenStackShort} {rhos_prev_ver}.
+* It is recommended to set the replica count of the `manilaAPI` service and the `manilaScheduler` service to 3. Ensure that you set the replica count of the `manilaShares` service to 1.
+* Ensure that the appropriate storage management network is specified in the `manilaShares` section. The example below connects the `manilaShares` instance with the CephFS back-end driver to the `storage` network.
+* Prior to adopting the `manilaShares` service for CephFS through NFS, ensure that you create a clustered Ceph NFS service. The name of the service must be `cephfs_nfs_cluster_id`.
 
-. Patch `OpenStackControlPlane` to deploy the {rhos_component_storage_file}; here's an example that uses
-Native CephFS:
+. Patch the `OpenStackControlPlane` CR to deploy the {rhos_component_storage_file}. The following example patch file uses native CephFS:
```

Comment:
Are steps 1-3 more like prerequisites? Does the "main" procedure actually start with patching the `OpenStackControlPlane` CR?